website/docs/r/disk_encryption_set.html.markdown

```diff
@@ -0,0 +1,140 @@
+---
+subcategory: "Compute"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_disk_encryption_set"
+sidebar_current: "docs-azurerm-resource-disk-encryption-set"
+description: |-
+  Manage Azure DiskEncryptionSet instance.
+---
+
+# azurerm_disk_encryption_set
+
+Manage an Azure DiskEncryptionSet instance.
+
+-> **NOTE:** The DiskEncryptionSet service is currently in public preview and are only available in a limited set of regions. 
+

```

Comment:
since this requires that the Key Vault is configured in a particular way - can we also document that here:

```

-> **NOTE:** At this time the Key Vault used to store the Active Key for this Disk Encryption Set must have both Soft Delete & Purge Protection enabled - which are not yet supported by Terraform - instead you can configure this using [a provisioner](https://www.terraform.io/docs/provisioners/local-exec.html) or [the `azurerm_template_deployment` resource](https://www.terraform.io/docs/providers/azurerm/r/template_deployment.html).

```

Reply:
Added. And do you think we should add the provisioner command in the example?